backend/src/libs/packages/open-ai/open-ai.package.ts

```diff
@@ -37,6 +37,14 @@ class OpenAI {
     return this.parseCompletionResponse(text);
   }
 
+  public async createImage(prompt: string): Promise<string> {
+    const response = await this.openAIApi.createImage({
+      prompt,
+    });
+
+    return response.data.data[0]?.url ?? '';
```

Comment:
```suggestion
    const { data } = await this.openAIApi.createImage({
      prompt,
    });
    
    const [image] = data.data;

    return image?.url ?? '';
```
but you need to get an buffer i guess to save it to the s3